### PR TITLE
docs: Added CPU requirements

### PR DESCRIPTION
**Pull request by gauravsitlani**
Signed-off-by: Gaurav Sitlani <gaurav@koor.tech>

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [x] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/development-flow.html#commit-structure).
- [x] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [x] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/development-flow.html#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: Documentation/Getting-Started/Prerequisites/prerequisites.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ and Rook is granted the required privileges (see below for more information).
 
 Kubernetes **v1.17** or higher is supported for the Ceph operator.
 
+## CPU Architecture  
+
+We only support `amd64 / x86_64` and `arm64` architecture.
```
Comment (travisn) [inline Documentation/Getting-Started/Prerequisites/prerequisites.md:14]: How about this wording? "Support" is more of a downstream or product term.
```suggestion
Architectures released are `amd64 / x86_64` and `arm64`.
```

Comment (gauravsitlani) [reply Documentation/Getting-Started/Prerequisites/prerequisites.md]: @travisn sure, sounds good.
